packages/components/file-uploader-input/src/file-uploader-input.js

```diff
@@ -113,29 +129,28 @@ customElementDefineHelper(
 		}
 
 		get helpText() {
-			const acceptedFileExtensions = this.acceptedFileExtensions
-				.map(ext => ext.toUpperCase())
-				.join(', ');
+			const helpTextList = [];
+
+			if (!this.hideFileTypeHelpText && this.acceptedFileExtensions) {
+				const acceptedFileExtensions = this.acceptedFileExtensions
+					.map(ext => ext.toUpperCase())
+					.join(', ');
+				helpTextList.push(`Allowed formats: ${acceptedFileExtensions}`);
+			}
+
+			if (
+				this.multiple & !this.hideMaxFileNumberHelpText &&
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Review this use of bitwise "&" operator; conditional "&&" might have been intended. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ABitwiseOperators)